inc/profile.class.php

```diff
@@ -908,6 +942,24 @@ function showFormManagement($openform = true, $closeform = true) {
       $matrix_options['title'] = __('Management');
       $this->displayRightsChoiceMatrix($rights, $matrix_options);
 
+      echo "<div>";
```

Comment:
To enhance display.
```suggestion
      echo "<div class='tab_cadre_fixehov'>";
```

![image](https://user-images.githubusercontent.com/33253653/74661658-aac2e680-5198-11ea-892c-76cf4894622a.png)